client/jpackage/Kurtama.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Kurtama
+Comment=Kurtama
+Exec=/opt/kurtama-client/bin/Kurtama
+Icon=/opt/kurtama-client/lib/Kurtama.png
+Terminal=false
+Type=Application
+Categories=Game
+MimeType=
```

Comment:
I don't know what this `MimeType` option is for, but I looked it up and it might just be removed. I don't like it being empty. But whatever.